spec/spec_helper_acceptance.rb

```diff
@@ -289,18 +289,8 @@ def enable_ha_validations(options = {})
   ha_node_count = ha_node_ips.size
 
   if enable_ha
-    # Confirm ODL OVSDB HA is enabled
-    if ha_node_count >=2
-      # Check for HA_NODE_COUNT >= 2
-      describe file('/opt/opendaylight/deploy/jolokia.xml') do
-      it { should be_file }
-      it { should be_owned_by 'odl' }
-      it { should be_grouped_into 'odl' }
-    end
-    else
-      # Check for HA_NODE_COUNT < 2
-      fail("Number of HA nodes less than 2: #{ha_node_count} and HA Enabled")
-    end
+    # Check for HA_NODE_COUNT < 2
+    fail("Number of HA nodes less than 2: #{ha_node_count} and HA Enabled")
```

Comment:
Same here, missing conditional. We're not actually calling this helper in the beaker test it turns out, so it's not causing a failure (`bundle exec rake cent_6test_dock` passes).